platform/ios/ios.cmake

```diff
@@ -7,7 +7,9 @@ endif()
 # Override default CMake NATIVE_ARCH_ACTUAL
 # https://gitlab.kitware.com/cmake/cmake/-/issues/20893
 # https://stackoverflow.com/a/22689917/5531400
-set(CMAKE_OSX_ARCHITECTURES "$(ARCHS_STANDARD)")
+if(CMAKE_GENERATOR STREQUAL Xcode)
+    set(CMAKE_OSX_ARCHITECTURES "$(ARCHS_STANDARD)")
+endif()
 set_target_properties(mbgl-core PROPERTIES XCODE_ATTRIBUTE_ONLY_ACTIVE_ARCH[variant=Debug] "YES")
```

Comment:
Should the `set_target_properties` also be in the `if` block? I see that the comment on stackoverflow mentions the `set_target_properties` thing... https://stackoverflow.com/a/22689917/5531400

Reply:
`XCODE_ATTRIBUTE_*` only matters for XCode.

Reply:
OK